src/index.js

```diff
@@ -102,13 +102,23 @@ function mergeExtensionConfig(targetVersion, mergeTarget, extensionConfig, modul
   return deepmerge(mergeTarget, toMerge);
 }
 
+/** @type {string} */
 const hyvaThemesConfig = basePath
   ? JSON.parse(fs.readFileSync(path.join(basePath, hyvaThemeJsonInModule)))
-  : false;
+  : '';
 
-const hyvaModuleDirs = hyvaThemesConfig && hyvaThemesConfig.extensions
-  ? Object.values(hyvaThemesConfig.extensions || []).map(module => path.join(basePath, module.src))
-  : [];
+/**
+ * Add the full path to each path in a array
+ *
+ * @param {string[]} paths - to add the basePath to
+ * @param {string} subPath - subpath from paths to use
```

Comment:
Let's call it `property` or `key` since `subpath` makes it sounds as if it will be added directly as part of the path.

Reply:
Technally it is the extra part of the path, e.g. `vendor/module/folder` + `src`, but I changed it to `key` 😉